contracts/lease/src/api/open/mod.rs

```diff
@@ -196,3 +289,55 @@ mod test_invariant {
         ));
     }
 }
+
+#[cfg(test)]
+mod test_position_spec {
+
+    use cosmwasm_std::{from_slice, StdError};
+    use currency::lpn::Usdc;
+    use finance::{coin::Coin, duration::Duration, liability::Liability, percent::Percent};
+
+    use super::PositionSpec;
+
+    type LpnCoin = Coin<Usdc>;
+
+    #[test]
+    fn new_valid() {
+        let liability = Liability::new(
+            Percent::from_percent(65),
+            Percent::from_percent(5),
+            Percent::from_percent(10),
+            Percent::from_percent(2),
+            Percent::from_percent(3),
+            Percent::from_percent(2),
+            Duration::from_hours(1),
+        );
+        let position_spec = PositionSpec::new(
+            liability,
+            LpnCoin::new(5000).into(),
+            LpnCoin::new(9000000).into(),
+        );
+
+        assert_load_ok(position_spec, br#"{"liability":{"initial":650,"healthy":700,"first_liq_warn":730,"second_liq_warn":750,"third_liq_warn":780,"max":800,"recalc_time":3600000000000},"min_asset":{"amount":"5000","ticker":"USDC"},"min_sell_asset":{"amount":"9000000","ticker":"USDC"}}"#);
+    }
+
+    #[test]
+    fn new_invalid() {
```

Comment:
I believe we can add a negative test for the min_asset.ticker != min_sell_asset.ticker